core/src/main/java/org/apache/accumulo/core/data/Key.java

```diff
@@ -657,6 +657,21 @@ public ByteSequence getRowData() {
     return new ArrayByteSequence(row);
   }
 
+  /**
+   * Writes the row ID into the given <code>ArrayByteSequence</code> without allocating new object
+   * by using {@link org.apache.accumulo.core.data.ArrayByteSequence#reset(byte[], int, int)}
+   * method. Using this updates existing ArrayByteSequence object with reference to row data, rather
+   * than copying row data.
+   *
+   * @param r <code>ArrayByteSequence</code> object to copy into
```

Comment:
I'm not sure why these javadocs use `<code>`, but it makes more sense to be using the javadoc tag for `{@link ArrayByteSequence}` instead, I think.